src/ray/protobuf/core_worker.proto

```diff
@@ -55,6 +55,12 @@ message ActorHandle {
 
   // How many times tasks may be retried on this actor if the actor fails.
   int64 max_task_retries = 9;
+
+  // The name of the actor.
+  string name = 10;
+
+  // The namespace that this actor belongs to.
+  string ray_namespace = 11;
```

Comment:
looks we are inconsistent about namespace vs ray_namespace; any reason why?

Reply:
The existing RPC messages seem to use ray_namespace, so I followed it. I think namespace was a reserved keyword or something (I tried namespace at first, and didn't work for some reasons).